noxfile.py

```diff
@@ -294,18 +294,8 @@ def install_systemtest_dependencies(session, *constraints):
 
 
 @nox.session(python=SYSTEM_TEST_PYTHON_VERSIONS)
-@nox.parametrize(
-    "protobuf_implementation,database_dialect",
-    [
-        ("python", "GOOGLE_STANDARD_SQL"),
-        ("python", "POSTGRESQL"),
-        ("upb", "GOOGLE_STANDARD_SQL"),
-        ("upb", "POSTGRESQL"),
-        ("cpp", "GOOGLE_STANDARD_SQL"),
-        ("cpp", "POSTGRESQL"),
-    ],
-)
-def system(session, protobuf_implementation, database_dialect):
```

Comment:
Why is this being removed?

Reply:
This was added because of some conflicts. Rebased the branch from main again and these changes were removed

Reply:
These changes should go away with https://github.com/googleapis/python-spanner/pull/1320